javascript_es6/1.a.browser-echobot/README.md

```diff
@@ -0,0 +1,42 @@
+The example shows the use of the botbuilder-js SDK for the browser using the [BotFramework-WebChat](https://github.com/Microsoft/BotFramework-WebChat) and a custom [WebChatAdapter](/src/webChatAdapter.js). After running the bot, to see it in action, visit `http://localhost:8080`
```

Comment:
This line scrolls off the side of my 4k monitor. Can you break it into 2 or 3 lines?